src/components/common.js

```diff
@@ -92,8 +92,8 @@ export const Clickable = forwardRefWithName('Clickable',
 export const Link = forwardRefWithName('Link', ({ disabled, variant, children, baseColor = colors.accent, ...props }, ref) => {
   return h(Clickable, _.merge({
     ref,
-    style: { // 0.72 is the min to meet ANDI's contrast requirement
-      color: disabled ? colors.dark(0.72) : baseColor(variant === 'light' ? 0.3 : 1),
+    style: { // Note: 0.72 is the min to meet ANDI's contrast requirement
+      color: disabled ? colors.disabled(0.5) : baseColor(variant === 'light' ? 0.3 : 1),
```

Comment:
I would expect the color of a disabled link to be `colors.disabled`, not another color derived from `colors.disabled`. I'd vote for defining `colors.disabled` as the result of this function.

Reply:
Do you have any suggestions for implementing that?
Unless we make some adjustments to how colors.js is used, we have to use the function to adjust the intensity of the color, or we hard code the resulting color (rgb(182, 183, 184) / #B7B8B6)

Seems like our implementation of colors don't directly match with design unless that resulting RGB is correct.

Reply:
I agree with what @nawatts wrote. I think we can implement that by having

```javascript
const baseColors = {
  disabled: '#b6b7b8' // instead of '#6d6e70'
}
```

in `colors.js` and then at the call site (in `common.js`):

```javascript
color: disabled ? colors.disabled() : baseColor(variant === 'light' ? 0.3 : 1),
```

instead of

```javascript
color: disabled ? colors.disabled(0.5) : baseColor(variant === 'light' ? 0.3 : 1),
```

Reply:
What @kyuksel said. If we want all disabled elements to be a certain color, let's set `colors.disabled` to that color instead of relying on every component to apply the same color mixing to a base color.